src/MCRun.jl

```diff
@@ -206,14 +226,14 @@ function atom_move!(mc_state::MCState, i_atom, pot, ensemble)
     #move randomly selected atom (obeying the boundary conditions)
     trial_pos = atom_displacement(mc_state.config.pos[i_atom], mc_state.max_displ[1], mc_state.config.bc)
     #find new distances of moved atom 
-    delta_en, dist2_new = energy_update(trial_pos, i_atom, mc_state.config, mc_state.dist2_mat, pot)
+    d_en_move, dist2_new = energy_update(trial_pos, i_atom, mc_state.config, mc_state.dist2_mat, pot)
```

Comment:
more redefining variable names

Reply:
There seemed to be a typo in line 186: ma**c**_acc instead of ma**x**_acc

again re-name, perhaps delta_en_move

Reply:
Done on both counts